portfolio/src/main/webapp/script.js

```diff
@@ -47,8 +53,25 @@ function createTaskElement(task) {
   taskElement.className = 'task';
 
   const titleElement = document.createElement('span');
+
   titleElement.innerText = task.comment + ' \n - Commented by ' + task.displayName;
 
   taskElement.appendChild(titleElement);
   return taskElement;
 }
+
+// async function loggedIn(){
```

Comment:
why is this here? if you need it then comment with multi line comment and leave a TODO saying why ex:
```
/* TODO: fix issue with login etc etc
  async function loggedIn() {
    ...
  }
*/
```